apps/web/app/(authenticated)/(app)/app/settings/root-keys/new/client.tsx

```diff
@@ -57,6 +57,16 @@ export const Client: React.FC<Props> = ({ apis }) => {
   const [showKey, setShowKey] = useState(false);
   const [showKeyInSnippet, setShowKeyInSnippet] = useState(false);
 
+  const handleSetChecked = useCallback((permission: UnkeyPermission, checked: boolean) => {
```

Comment:
minimal value. just passing a memoized fn to the PermissionToggle child component. But actually, since `handleSetChecked` has zero deps, its actually insignificant. Will remove it 😅

Reply:
Done! Since the function isn’t that expensive to recreate (zero deps lol), using useCallback might not yield significant performance benefits.

Reply:
cool, yeah that's what I thought too